pkg/hubble/parser/threefour/parser.go

```diff
@@ -97,8 +97,7 @@ func (p *Parser) Decode(data []byte, decoded *pb.Flow) error {
 	}
 
 	var packetOffset int
-	var eventType uint8
-	eventType = data[0]
+	var eventType uint8 = data[0]
```

Comment:
Even simpler would be:
`eventType := data[0]`

Reply:
Applied, thanks!